components/gc-follow-us/_base.scss

```diff
@@ -280,13 +279,6 @@ $social-media-icons-size: 38px;
 		@extend %social-media-icons-logo-instagram;
 	}
 
-	.googleplus { //This is deprecated and will be removed from later versions
-		@extend %social-media-icons-followus-logo-properties;
-		@extend %social-media-icons-logo-googleplus;
-		background-repeat: no-repeat;
-		background-size: 35px 35px;
-	}
```

Comment:
Can you duplicate the code about "googleplus" into the "deprecated" méli-mélo project. So for the one that want to use it, they will be able to use the méli-mélo compilation.

I know that was already mentioned in the code, but it was never officially communicated. Thanks